Casks/sol.rb

```diff
@@ -0,0 +1,24 @@
+cask "sol" do
+  version "1.1.20"
+  sha256 :no_check
+
+  url "https://github.com/ospfranco/sol/releases/latest/download/Sol.zip"
+  name "Sol"
+  desc "Creative Open Source Launcher"
```

Comment:
I think this captures the usage more
```suggestion
  desc "Launcher & command palette"
```